Source/WebKit/UIProcess/ios/WKMouseInteraction.h

```diff
@@ -23,22 +23,33 @@
 * THE POSSIBILITY OF SUCH DAMAGE.
 */
 
+#pragma once
+
 #if HAVE(UIKIT_WITH_MOUSE_SUPPORT)
 
-#import <UIKit/UIHoverGestureRecognizer.h>
 #import "NativeWebMouseEvent.h"
-#import "UIKitSPI.h"
+#import <UIKit/UIKit.h>
+
+@class WKMouseInteraction;
+
+@protocol WKMouseInteractionDelegate<NSObject>
+- (void)mouseInteraction:(WKMouseInteraction *)interaction changedWithEvent:(std::unique_ptr<WebKit::NativeWebMouseEvent>&&)event;
```

Comment:
We're not sinking `event` here, why not just pass by const l-value ref?

Reply:
That's a good point! Fixed